src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/MediaApplication.cshtml

```diff
@@ -170,14 +170,19 @@
             <div class="modal-body">
                 <div class="form-group">
                     <label for="input" class="form-control-label" id="input-label">@T["Folder name"]</label>
-                    <input type="text" id="create-folder-name" class="form-control">
+                    <input type="text" id="create-folder-name" class="form-control d-none d-sm-block">
+                    <input type="text" id="create-folder-name" class="form-control form-control-sm d-block d-sm-none">
                 </div>
                 <div id="createFolderModal-errors"></div>
             </div>
-            <div class="modal-footer">
+            <div class="modal-footer d-none d-sm-block">
                 <button type="button" class="btn btn-primary" id="modalFooterOk">@T["Ok"]</button>
                 <button type="button" class="btn btn-light" data-dismiss="modal">@T["Cancel"]</button>
             </div>
+            <div class="modal-footer d-block d-sm-none">
+                <button type="button" class="btn btn-sm btn-primary" id="modalFooterOk">@T["Ok"]</button>
```

Comment:
The same here.
Clicking the "OK" button is not working anymore.

Having two inputs with the same id breaks the 'onclick' handler that we have here:
https://github.com/OrchardCMS/OrchardCore/blob/cbd7bdfb9373dabb499f9122cf4cf0d196103d10/src/OrchardCore.Modules/OrchardCore.Media/Assets/js/MediaApp/app.js#L427

You can take a look at : 
https://api.jquery.com/id-selector/ 
Where it says that "If more than one element has been assigned the same ID, queries that use that ID will only select the first matched element in the DOM"

Reply:
Please have a look to my above comment, also I will verify before I make the PR ready

Reply:
Yes don't add a second modal element if we can use Bootstrap classes to alter it's style depending on breakpoints.

Reply:
Bootstrap classes will control showing or hiding the elements